### ISSUE TITLE
[unity] Visual artifacts in Unity on all my rigs - new and old

### ISSUE THREAD

**Issue by TowerCrow** [EsotericSoftware/spine-runtimes#1510]
More info here: http://esotericsoftware.com/forum/unity-Weird-artifacts-on-my-latest-rig-in-Unity-12835

**Comment by TowerCrow** [comment]
Emailed a reproduction project to you guys.

**Comment by badlogic** [comment]
Cheers! @HaraldCsaszar repro is in email FMfcgxwDrbsZnTtkXNZDCBdkXFctGgSB

**Comment by HaraldCsaszar** [comment]
Cause of the observed problem:
All textures have been exported as `Premultiplied alpha` textures, but the imported texture settings in Unity all have the parameter `Advanced - Alpha is Transparency` enabled, which is wrong and causes visual artifacts.

Correct import settings are documented here, for both `premultiplied alpha` (default) and `straight alpha` workflows:
http://esotericsoftware.com/spine-unity#Advanced---Premultiplied-vs-Straight-Alpha-Import

Solution: select all spine atlas textures (e.g. via filter "t:texture") and disabled the  `Advanced - Alpha is Transparency` parameter.

**Comment by TowerCrow** [comment]
The project I emailed you is completely fresh with no other assets present. I created a new project. Imported latest runtimes, deleted examples folder and dropped in my rigs. So it's the default behaviour for some reason. I could reproduce it in two other projects I have as well. The export settings in Spine are also the Defaults, I specifically made sure of that. 

I didn't think of checking the import settings because previously this exact worklfow worked fine, now it doesn't. Hence why I'm posting. Furthermore, rigs that I exported months ago and worked fine previously have this same exact issue when freshly imported in a project with latest runtimes.

But it's easy to fix, so thank you for the information. I'll know what to do when this happens next time.
